<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEditMandatoryGroup</class>
 <widget class="QDialog" name="CEditMandatoryGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="NameLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="GroupName_Label">
       <property name="text">
        <string>Group name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="GroupName_Edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="MandatoryMark_Label">
       <property name="text">
        <string>Mandate label:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="MandatoryMark_comboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="Rights_groupBox">
     <property name="title">
      <string>Rights</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QCheckBox" name="alertsView_checkBox">
        <property name="text">
         <string>View access journal </string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="alertsDelete_checkBox">
        <property name="text">
         <string>Clear access journal</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="generalSettingsView_checkBox">
        <property name="text">
         <string>View general settings</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="UserControl_checkBox">
        <property name="text">
         <string>Users and groups setup</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="DeleteDevice_checkBox">
        <property name="text">
         <string>Remove devices</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="generalSettingsChange_checkBox">
        <property name="text">
         <string>Edit general settings</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="editDevice_checkBox">
        <property name="text">
         <string>Edit devices</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="VideosDecrypt_checkBox">
        <property name="text">
         <string>Export media</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="VideosView_checkBox">
        <property name="text">
         <string>View media journal</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="AddDevice_checkBox">
        <property name="text">
         <string>Add devices</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="videosDelete_checkBox">
        <property name="text">
         <string>Delete media</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
